terraform/modules/s3-bucket/main.tf

```diff
@@ -0,0 +1,15 @@
+resource "aws_s3_bucket" "s3_bucket" {
+  bucket = "${var.bucket_name}"
+  acl    = "private"
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [{
+    "Effect": "Allow",
+    "Principal": {"AWS": "arn:aws:iam::${var.access_account_id}:root"},
+    "Action": "*",
```

Comment:
Yeah they're the ones. So does this mean that you can ListBucket and GetBucketLocation from the aws cli? Or the console as well? And if the bucket was only going to be used via the apps, would you still need these permissions; if the apps know which bucket to use is being able to Delete/Get/PutObject enough?